client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -244,6 +274,7 @@ export const SitesEllipsisMenu = ( {
 					<ManagePluginsItem { ...props } />
 					{ ! isNotAtomicJetpack( site ) && <HostingConfigItem { ...props } /> }
 					{ site.is_coming_soon && <PreviewSiteModalItem { ...props } /> }
+					{ isEnabled( 'sites/copy-site' ) && <CopySiteItem { ...props } /> }
```

Comment:
Can we make sure this is only present for the site owner too?

Reply:
The check is inside the component.
https://github.com/Automattic/wp-calypso/blob/24545be918bc9b544f4f3e70e997468056cfa106/client/sites-dashboard/components/sites-ellipsis-menu.tsx#L190-L192